frontend-react/src/components/Table/TableFilters.tsx

```diff
@@ -27,10 +27,8 @@ interface SubmissionFilterProps {
 /* This helper ensures start range values are inclusive
  * of the day set in the date picker. */
 const inclusiveDateString = (originalDate: string) => {
```

Comment:
I realize this existed before, but when casting between types, it's nice to use a bit of Hungarian notation so it's clear what the type currently is.  Like `originalDateStr`/`originalDateString` in this case since `originalDate` is a little misleading 🤷 

Not a huge deal at all, just nice for readability 🤷 🤷 🤷